### ISSUE TITLE
Fixed week lines count

### ISSUE THREAD

**Issue by VladimirMitso** [aleksanderwozniak/table_calendar#239]
Hi.
I want to implement 6 visible weeks in every month.
Something like this.
![sample](https://user-images.githubusercontent.com/31623922/79981766-e5edd880-84ad-11ea-9f52-c44de7d9e39f.jpg)
Is it possible without changing `_daysInMonth` method in `calendar_controller.dart` class?
```
List<DateTime> _daysInMonth(DateTime month) {
    final first = _firstDayOfMonth(month);
    final daysBefore = _getDaysBefore(first);
    final firstToDisplay = first.subtract(Duration(days: daysBefore));

    final last = _lastDayOfMonth(month);
    final daysAfter = _getDaysAfter(last);

    final lastToDisplay = last.add(Duration(days: daysAfter));
    return _daysInRange(firstToDisplay, lastToDisplay).toList();
  }
```

**Comment by VladimirMitso** [comment]
PR made https://github.com/aleksanderwozniak/table_calendar/pull/240

**Comment by aleksanderwozniak** [comment]
This has now landed with `3.0.0-beta` version. To use it, set `sixWeekMonthsEnforced` property to true.

**Comment by ebot64** [comment]
Is there a way I can force the calendar to display 5 weeks irrespective of months that spill into 6 weeks?
